test_runner/fixtures/compare_fixtures.py

```diff
@@ -105,6 +105,8 @@ def __init__(
         self._pg_bin = pg_bin
         self.pageserver_http_client = self.env.pageserver.http_client()
 
+        # note that neon_simple_env now uses LOCAL_FS remote storage
```

Comment:
nit: this comment probably won't make a lot of sense to future readers (they don't know what "now" is unless they're looking at git blame)

Reply:
This fixture is used by the benches, so I was thinking it might quickly stop the troubleshooting of "why is pageserver uploading stuff during my benchmark". Do you have recommendations?

Reply:
Something like `# neon_simple_env uses LOCAL_FS remote storage by default, so although benchmarks mainly target local storage, you may see uploads to the LOCAL_FS backend"` perhaps?

Not a big deal either way.

Reply:
Oki, just leaving this.